truckersmp-cli

```diff
@@ -529,6 +531,25 @@ Please report an issue: {}""".format(e, URL.issueurl))
             sys.exit("Failed to download mod files.")
 
 
+def get_current_steam_user():
+    """
+    Get the current AccountName with saved login credentials.
+
+    If successful this returns the AccountName of the user with saved credentials.
+    Otherwise this returns None.
+    """
+    for path in File.loginusers_paths:
+        try:
+            login_vdf = vdf.parse(open(path))
+
+            for _, info in login_vdf["users"].items():
```

Comment:
In this case (`_` is unused), it's better to use `values()` method instead of `items()`:
```python
#!/usr/bin/env python3

import vdf

with open("/path/to/steam/installation/config/loginusers.vdf") as f:
    login_vdf = vdf.parse(f)
    for user in login_vdf["users"].values():
        if user["mostrecent"] == "1" and user["RememberPassword"] == "1":
            print(user["AccountName"])
```